utils/unittest/fixtures.go

```diff
@@ -1040,7 +1058,7 @@ func EpochStatusFixture() *flow.EpochStatus {
 
 func IndexFixture() *flow.Index {
 	return &flow.Index{
-		NodeIDs:       IdentifierListFixture(5),
+		NodeIDs:       nil,
```

Comment:
I am in favour of removing this field. But if we keep it, we should populate it because otherwise, we might be concealing errors (like this field not being persisted on disk).